deployer/__main__.py

```diff
@@ -19,6 +18,26 @@
 yaml = YAML(typ="safe", pure=True)
 
 
+def auth(cluster_name):
+    """
+    Quickly gain command-line access to a cluster using deployer credentials
+    """
```

Comment:
Can you add to this docstring a bit? Some questions left unanswered to me are:

- Is this a "call once at any time to update the current kubeconfig file with credentials to a k8s cluster"?
- Is it supposed to be used as a CLI command only, or as a helper by the python code overall?